tests/by-util/test_cp.rs

```diff
@@ -497,6 +497,45 @@ fn test_cp_strip_trailing_slashes() {
     assert_eq!(at.read(TEST_HELLO_WORLD_DEST), "Hello, World!\n");
 }
 
+#[test]
+fn test_cp_parents() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    let result = ucmd
+        .arg("--parents")
+        .arg(TEST_COPY_FROM_FOLDER_FILE)
```

Comment:
sorry but could you please add a new test when copying two files into a directory with this option?
thanks

Reply:
No problem! I added a test that copied a file in a directory and one in the pwd. Would it be a better idea to touch a couple of files in a couple of different directories and test copying them to a new directory? I suppose in that case we would just check it the copy was successful?

Thanks for the reviews!